cmd/text.go

```diff
@@ -23,44 +23,48 @@ package cmd
 const (
 	// Param 1: cookbook directory in repository.
 	// Param 2: newline-separated list of cookbooks
-	CookbooksNotSourcedTxt = `Changes made to the following cookbooks in %s
+	CookbooksNotSourcedTxt = `
+------------------------ WARNING ---------------------------
+Changes made to the following cookbooks in %s
 cannot be saved upstream, though they can still be uploaded
 to a Chef Server:
 
 %s
 
+-----------------------------------------------------------
 `
-	CookbookCaptureCompleteTxt = `You're ready to begin!
+	CookbookCaptureCompleteTxt = `
+You're ready to begin!
 
 Start with 'kitchen converge'.  As you identify issues, you
 can modify cookbooks in their original checkout locations or
 in the repository's cookbooks directory and they will be picked
 up on subsequent runs of 'kitchen converge'.
 `
 	// Param 1: repository directory
-	// Param 2: newline-separated space-indented list of cookbooks
 	CookbookCaptureGatherSourcesTxt = `
 Repository has been created in '%s'.
 
-Please clone or check out the following cookbooks locally
-from their original sources, and provide the base path
-for the checkout:
-
-%s
+Next, locate version-controlled copies of the cookbooks. This is
+important so that you can track changes to the cookbooks as you
+edit them. You may have one or more existing paths where you have
+checked out cookbooks. If not, now is a good time to open a
+separate terminal and clone or check out the cookbooks.
 
 If all cookbooks are not available in the same base location,
 you will have a chance to provide additional locations.
 
-If sources are not available for these cookbooks, leave this blank.
-
-Checkout Location [none]: `
+Press Enter to Continue:`
```

Comment:
I thought maybe `*Press Enter to Continue*` because we're not asking the user to enter text, but I think the `:` for "waiting for your input" makes as much sense. Thoughts? I could go either way

Reply:
I was also thinking it would be nice to ignore any characters the user types besides the return key, but I don't see a simple way with `Scan` to do that, so I think this is fine